### ISSUE TITLE
Download quota has been exceeded for the rgb-frontal-veiw videos

### ISSUE THREAD

**Issue by Eddie-Hwang** [how2sign/how2sign.github.io#8]
Hi,

Thank you for providing a large scale of sign language dataset.
However, I found that some of files exceeded the download quota and are not available.
Can you please fix this issue?

Thank you in advance.

**Comment by how2sign** [comment]
Hi Eddie, 
thank you for letting us know. 
This limit is usually by day, so if you try again today, it most probably will work.
But if you let me know which files from the RGB-frontal-view folder you had problems downloading I can verify. 
I just test it the first 2 files and it was working from my side.
Let me know in case you still have problems downloading it.

Best,

**Comment by Eddie-Hwang** [comment]
Hi,
It skips all download processes (e.g. z01) when using download shell scripts.
I have used alternative way using gdown. Then I got a following error.

```
gdown https://drive.google.com/uc?id=1lLu4CyDfkmDpsKGh9TDXt17WnlPb09jS
Access denied with the following error:

 	Too many users have viewed or downloaded this file recently. Please
	try accessing the file again later. If the file you are trying to
	access is particularly large or is shared with many people, it may
	take up to 24 hours to be able to view or download the file. If you
	still can't access a file after 24 hours, contact your domain
	administrator. 

You may still be able to access the file from the browser:

	 https://drive.google.com/uc?id=1lLu4CyDfkmDpsKGh9TDXt17WnlPb09jS
```

**Comment by rabeya-akter** [comment]
The links aren't working right now.
![image](https://github.com/how2sign/how2sign.github.io/assets/44209638/e69d39e0-6fd9-4416-aad0-525db9573eac)

**Comment by 2000ZRL** [comment]
> Hi, It skips all download processes (e.g. z01) when using the download shell script. I have used an alternative way using `gdown`. Then I got a following error.
> 
> ```
> gdown https://drive.google.com/uc?id=1lLu4CyDfkmDpsKGh9TDXt17WnlPb09jS
> Access denied with the following error:
> 
>  	Too many users have viewed or downloaded this file recently. Please
> 	try accessing the file again later. If the file you are trying to
> 	access is particularly large or is shared with many people, it may
> 	take up to 24 hours to be able to view or download the file. If you
> 	still can't access a file after 24 hours, contact your domain
> 	administrator. 
> 
> You may still be able to access the file from the browser:
> 
> 	 https://drive.google.com/uc?id=1lLu4CyDfkmDpsKGh9TDXt17WnlPb09jS
> ```

Yes, I meet the same issue. The given .sh script doesn't work, and gdown also cannot retrieve the link..

**Comment by cleong110** [comment]
See also https://github.com/how2sign/how2sign.github.io/issues/17
